tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/endpoints/EndpointTests.java

```diff
@@ -67,9 +67,12 @@ public String getAsCppSourceString(boolean isEndpointProperty, int indent) throw
             for (Map.Entry<String, EndpointTestParameter> paramEntry : this.entrySet()) {
                 builder.append(separator);
                 builder.append(paramEntry.getValue().getAsCppSourceString(isEndpointProperty));
-                if (builder.length() > onPreviousLinesLength + 100) {
+                if (builder.length() > onPreviousLinesLength + 150) {
                     onPreviousLinesLength = builder.length();
-                    separator = ",\n" + String.join("", Collections.nCopies(indent + 1, " "));;
+                    if (isEndpointProperty)
+                        separator = ",\n" + String.join("", Collections.nCopies(indent + 3, " "));
```

Comment:
something to explore for later: doing all those indent calculations is so messy. can we generate badly formatted code and just run it through formatter?

Reply:
frankly speaking I though about just dumping a json and then parsing it in runtime.
But for now just diving deeper into a rabbit hole.